CollAction/Views/Home/FAQ.cshtml

```diff
@@ -281,5 +281,20 @@
             </div>
         </div>
     </div>
+
+    <div class="panel panel-default">
+        <div class="panel-heading" data-toggle="collapse" href="#collapseHowKnow">
+            <h3 class="panel-title">
+                <span class="fa fa-rocket" aria-hidden="true"></span>&nbsp;
+                When will the projets end exactly?
+                <span class="fa fa-angle-down pull-right"></span>
+            </h3>
+        </div>
+        <div id="collapseHowKnow" class="panel-collapse collapse">
+            <div class="panel-body">
+                The project ends at the end (23:59:59) of the day that is listed as "End Date" in the UTC (London) timezone.
```

Comment:
This needs to be translaten to dutch.

It's: "Projecten eindigen aan het einde van de dag (23:59:59) in de UTC (Londen) tijdzone van de einddatum van het project."